fair-acc/opendigitizer
# Frank/metadata propagation

Adapt to latest graph-prototype: 
 - Use delay in test block to receive all data.
 - Use messages for scheduler control/communication in GnuRadioWorker (simplifies the service<->scheduler thread interaction).

Expand tests to ensure that metadata configured at the GR sources (in GRC) make it to the data sink and from there to the REST client.
